### PR TITLE
Features/mobile intro component

### PR DESCRIPTION
**Pull request by Diegopariza**
Here you can see the changes that I do to the project when I create the component Intro:

- It creates "intro.js" inside the folder "src/js/components/intro.js".
- It creates "intro.scss" inside the folder "src/scss/components/intro.scss".
- It adds the images "FormaCirculoAzul.png" and "FormaNaranja.png" to
"src/static/assets/FormaCirculoAzul.png - FormaNaranja.png".
- In "intro.js" it creates the function "nodeImages" that contains the object "imagesArrays" from the data, this object takes the 'imagesArrays' array and place a map on it that creates an image per element of the array and places a class and a modifier that changes according to the number of elements called and returns an array of image nodes.
- In "intro.js" it creates the function "componentIntro" that contains the object "data" this object have the information of the components.
- Link of the task in Trello: https://trello.com/c/iKuBgJnO
- Link of the used desing in Zeplin: https://app.zeplin.io/project/5f00e6fda64d6e11dca0753a/screen/5f00e7cd0312ce95b07caa0d

### CODE REVIEW

File: src/contacto.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
```
Comment (camilomejiag) [inline src/contacto.html:1]: Why did you remove the indentation inside the `html` tag?

File: src/scss/components/intro.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    font-size: $mobile-paragraph;
+    height: 88px;
+    margin-left: 49px;
+    margin-right: 49px;
+    margin-top: 27px;
+    width: 270px;
+  }
+}
+
+
```
Comment (camilomejiag) [inline src/scss/components/intro.scss]: Remove one of the extra lines in the end of this file

File: src/scss/layout/_navigation.scss [hunk 3]
```diff
@@ -51,6 +51,7 @@
     transform: translateY(0);
   }
 
+
```
Comment (camilomejiag) [inline src/scss/layout/_navigation.scss]: This extra line can be removed, so its consistent with the rest of the file

File: src/js/components/intro.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  const imageIntro = nodeImages(data.images);
+  // eslint-disable-next-line prettier/prettier
+  const containerGeneral = newContainer('div', [...imageIntro, title, paragraph], ['intro__container-general']);
+  const section = newContainer('section', [containerGeneral], ['intro']);
+
+  return section;
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export { componentIntro };
```
Comment (camilomejiag) [inline src/js/components/intro.js:33]: And an extra line at the end of this file

File: src/js/components/intro.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import { newText, newMultimedia, newContainer } from './helpers';
+
+/**
+ * @param {Array [objectImages]} imagesArrays es un array de imagenes de la data
```
Comment (camilomejiag) [inline src/js/components/intro.js]: Usually, comments are used to *briefly* explain what something does, and the expected outcome. The process is documented if necessary. The idea is to make code that explains itself, and compliment some complex/weird behaviors with comment.

In this case, the code is clear enough to understand what its making, but comments are always welcome. The comment is longer than the code itself. Maybe you can summarize the behavior like:
```
This function receives an array of object and return a list of images
```
(Not the best example of a comment, but I hope you get the point) 
😄

File: src/contacto.html (hunk 1 above)
Comment (Diegopariza) [reply src/contacto.html:1]: Camilo, I already add the indentation to the code, can you please tell me if this is what you report me?

image

File: src/scss/components/intro.scss (hunk 2 above)
Comment (Diegopariza) [reply src/scss/components/intro.scss]: Camilo, thanks for your check, the errors that you report me was already fixed.

File: src/scss/layout/_navigation.scss (hunk 3 above)
Comment (Diegopariza) [reply src/scss/layout/_navigation.scss]: Camilo, thanks for your check, the errors that you report me was already fixed.

File: src/js/components/intro.js (hunk 4 above)
Comment (Diegopariza) [reply src/js/components/intro.js:33]: Camilo, thanks for your check, the errors that you report me was already fixed.

File: src/scss/components/intro.scss [hunk 6]
```diff
@@ -0,0 +1,55 @@
+.intro {
+  background-image: url('../static/assets/blue-form.png');
+  background-position-x: right;
+  background-position-y: 254px;
+  background-repeat: no-repeat;
+
+  &__container-general {
+    align-items: center;
```
Comment (caritonaranjo) [inline src/scss/components/intro.scss:8]: when I inspect you component, this align moves the pictures in the wrong position.

Captura de pantalla de 2020-08-13 22-08-56
Captura de pantalla de 2020-08-13 22-08-52

Comment (Diegopariza) [reply src/scss/components/intro.scss:8]: I verify the bug with Camila and we show everthing normal in her pc, please check again and tell me if the issue persist.

**Review by juang961225 — APPROVED** [review]

**Review by caritonaranjo — APPROVED** [review]
